linux/src/async_socket_linux.c

```diff
@@ -109,6 +110,13 @@ static int thread_worker_func(void* parameter)
     {
         struct epoll_event events[MAX_EVENTS_NUM];
         int num_ready = epoll_wait(g_epoll, events, MAX_EVENTS_NUM, EVENTS_TIMEOUT_MS);
+        if (num_ready == -1)
+        {
+            char err_msg[128] = {0};
+            (void)strerror_r(errno, err_msg, 128);
+            LogError("Failure epoll wait. Error: %s", MU_P_OR_NULL(err_msg));
```

Comment:
>LogError [](http://example.com/codeflow?start=12&length=8)

just in care strerror fails... it should print errno too.

Reply:
Maybe it cannot fail... on a second reading of documentation.

Reply:
Yes, it shouldn't fail, but having the errno is not a bad idea.